File: ConvertTheTemperature/ans.php
<?php

/**
 * @param Float $celsius
 * @return Float[]
 */
function convertTemperature($celsius) {
    $celvin = $celsius + 273.15;
    $fahrenheit = ($celsius * 1.80) + 32.00;

return [$celvin,$fahrenheit];
}

File: ConvertTheTemperature/ans.go
package main

import "fmt"

func convertTemperature(celsius float64) []float64 {
	return []float64{
		celsius + 273.15,
		celsius*1.80 + 32.00,
	}
}

func main() {
	fmt.Print(convertTemperature(36.50))
}
